### PR TITLE
Override present and present_with_time

### PR DESCRIPTION
**Pull request by JasonLG1979**
This should make sure our position is never lost.

### CODE REVIEW

**Comment by TingPing** [comment]
Can probably remove our other call to `self.set_initial_pos()` on Window creation then?

**Comment by TingPing** [comment]
Also maybe rename that function to `restore_position()` or something now.

**Comment by JasonLG1979** [comment]
> Can probably remove our other call to self.set_initial_pos() on Window creation then?



> Also maybe rename that function to restore_position() or something now.

Ok, sounds good. I'll amend the PR.

**Comment by JasonLG1979** [comment]
Done.

File: pithos/plugins/notification_icon.py [hunk 1]
```diff
@@ -194,11 +194,10 @@ def song_changed(self, window, song):
             self.statusicon.set_tooltip_text("{} by {}".format(song.title, song.artist))
 
     def _toggle_visible(self, *args):
-        self.window.set_visible(not self.window.get_visible())
-
-        if self.window.get_visible(): # Ensure it's on top
+        if self.window.get_visible():
+            self.window.hide_on_delete()
```
Comment (TingPing) [inline pithos/plugins/notification_icon.py]: Just use `self.window.hide()`. That method is only a convenience thing for directly connecting signals.

Comment (JasonLG1979) [inline pithos/plugins/notification_icon.py]: Done.

**Pull request merged**